Linux/sgx/test_app/sgx_t.mk

```diff
@@ -152,15 +151,15 @@ $(ENCLAVE_DIR)/TestEnclave_t.o: $(ENCLAVE_DIR)/TestEnclave_t.c
 	$(VCC) $(TestEnclave_C_Flags) -c $< -o $@
 	@echo "CC   <=  $<"
 
-$(ENCLAVE_DIR)/%.o: $(ENCLAVE_DIR)/%.cpp
+$(ENCLAVE_DIR)/%.o: $(ENCLAVE_DIR)/%.cpp $(ENCLAVE_DIR)/TestEnclave_t.o
```

Comment:
Why use _t.o?  Suppose the dependency is the generated header file. So using _t.c is better.

Reply:
I agree with you. Updated.